interview-questions.md

```diff
@@ -7,44 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: create a migration file to add foreign id(student_id) column. Or drop the whole table and start over.  
 
-  Researched answer:
+  Researched answer: Generate a column in your controller file then migrated so that it shows up in your schema. Delete the model and start from scratch adding your inheritance relations to decide wich is the primary key and where will the foreign key be assigned. 
```

Comment:
Not sure if you are serious about dropping the table and deleting the model.  Migration would be the way to go to fix this problem.  Starting over is a very last resort, and if you are working with a larger code base, may not be an option.